perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_attribute_def_def_o365EmailAddresses_mu.java

```diff
@@ -89,25 +102,33 @@ public void checkAttributeSemantics(PerunSessionImpl sess, Member member, Attrib
 		}
 		String ucoEmail = UCO + "@muni.cz";
 		if (!emails.contains(ucoEmail)) {
-			throw new WrongAttributeValueException(attribute, member, "does not contain " + ucoEmail);
+			throw new WrongReferenceAttributeValueException(attribute, attrUCO, member, null, "does not contain " + ucoEmail);
```

Comment:
You should add reference attribute holder too. It is the same member.

Reply:
Attribute reference holder added.